src/network/networkd-link.c

```diff
@@ -148,7 +148,9 @@ bool link_ipv4ll_enabled(Link *link, AddressFamilyBoolean mask) {
         if (!link->network)
                 return false;
 
-        if (STRPTR_IN_SET(link->kind, "vrf", "wireguard", "ipip", "gre", "ip6gre", "ip6tnl", "sit", "vti", "vti6", "can", "vcan", "vxcan", "nlmon"))
+        if (STRPTR_IN_SET(link->kind,
+                        "vrf", "wireguard", "ipip", "gre", "ip6gre","ip6tnl", "sit", "vti",
+                        "vti6", "can", "vcan", "vxcan", "nlmon", "xfrm"))
```

Comment:
We usually indent lines as aligned with '('. So, in this case, please add two more whitespace.